demo/vagrant/Vagrantfile

```diff
@@ -58,6 +58,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         node.vm.synced_folder "../../release", "/release", id: "release", :nfs => true, :mount_options => ['nolock,vers=3,tcp']
       end
 
+      # copy ceph config override file
+      node.vm.provision "file", source: "ceph-override.conf", destination: "/tmp/rook-conf/ceph-override.conf"
```

Comment:
actually changed the whole approach to use `write_files` directive in the cloud config.  i think it's pretty clean now.